### PR TITLE
Hmt 36 about requirements prizes

### PR DESCRIPTION
**Pull request by justin-phxm**
image

### CODE REVIEW

File: src/components/LandingPage/HackathonInformationContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  return (
+    <div
+      className={`flex w-full flex-row items-center justify-between gap-16 ${bgColor} ${fontColor} px-[5%] py-20`}
+    >
+      <div className={`flex flex-col gap-4 ${fontColor}`}>
+        {title}
+        <p className=" max-w-lg font-medium">{blurb}</p>
+      </div>
+      <div
+        className={`relative ${leftPosition && "order-first"} box-content hidden aspect-square w-2/3 min-w-48 max-w-96 rounded-lg md:block`}
```
Comment (ideen1) [inline src/components/LandingPage/HackathonInformationContainer.tsx:27]: Can you use the tailwind-merge library for this? Syntax would be like: `twMerge(....., leftPosition && "order-first", ...)`

**Review by ideen1 — COMMENTED** [review]
All looks good to me, just a small thing with tailwind class names

File: src/components/LandingPage/HackathonInformationContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+  title: React.ReactNode;
+  sectionImage: string;
+  leftPosition?: boolean;
+  blurb: string;
+  bgColor?: string;
+  fontColor?: string;
+}) {
+  return (
+    <div
+      className={`flex w-full flex-row items-center justify-between gap-16 ${bgColor} ${fontColor} px-[5%] py-20`}
```
Comment (ideen1) [inline src/components/LandingPage/HackathonInformationContainer.tsx:20]: Same thing here with tailwind-merge

File: src/components/LandingPage/HackathonInformationContainer.tsx (hunk 1 above)
Comment (justin-phxm) [reply src/components/LandingPage/HackathonInformationContainer.tsx:27]: The twMerge library is giving me a lot of eslint errors and I can't seem to figure out how to fix it. Might need some help on this one

**Review by ideen1 — APPROVED** [review]
LGTM!

**Comment by ideen1** [comment]
@justin-phxm Just bringing this to the top of your list since it has been approved. Just needs some merge conflict fixes

**Pull request merged**